src/librustdoc/html/static/css/rustdoc.css

```diff
@@ -1110,7 +1110,7 @@ table,
 	padding: 0 20px 20px 17px;
 }
 
-.item-info .stab {
+.item-info,.methods .stab {
```

Comment:
I think it's because of this change.

Reply:
This was indeed the case. I couldn't recall why I added this, so I've just removed it back 😅